### PR TITLE
formatting n images fixes

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+:::note
+This authorization code will expire within 1 hour. Please setup your Sumo source before it expires.
+:::
+
+1. In a browser where you are already authenticated to your Azure portal at https://portal.azure.com, navigate to the URL below, but replace `{{client_id}}` with your `Application (client) ID` from the earlier steps.
+```
+https://login.windows.net/common/oauth2/authorize?response_type=code&resource=https://outlook.office365.com&client_id={{client_id}}&redirect_uri=https://localhost
+```
+2. If successful, you will be redirected to a blank page with `code` and `session_state` as URL parameters. Write down the `code` value as this is your Oauth 2.0 authorization code you will use to configure the Sumo Logic C2C. This code is over 700 characters in length. Be careful to not copy the `&session_state=` at the end of the code. See below for an example:
+
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:100]:
```suggestion
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+### Oauth 2.0 Generate Authorization Code
+
+:::note
+This authorization code will expire within 1 hour. Please setup your Sumo source before it expires.
+:::
+
+1. In a browser where you are already authenticated to your Azure portal at https://portal.azure.com, navigate to the URL below, but replace `{{client_id}}` with your `Application (client) ID` from the earlier steps.
+```
+https://login.windows.net/common/oauth2/authorize?response_type=code&resource=https://outlook.office365.com&client_id={{client_id}}&redirect_uri=https://localhost
+```
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:98]:
```suggestion
1. In a browser where you are already authenticated to your Azure portal at https://portal.azure.com, navigate to the URL below, but replace `{{client_id}}` with your `Application (client) ID` from the earlier steps.
  ```
  https://login.windows.net/common/oauth2/authorize?response_type=code&resource=https://outlook.office365.com&client_id={{client_id}}&redirect_uri=https://localhost
  ```
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+10. You should now see both permissions, but you will need to click **Grant admin consent** to finalize the API permissions and confirm it. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-10-a.png')} alt="ms-exchange-api-perm-step-10-a.png" width="800" /> <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-10-b.png')} alt="ms-exchange-api-perm-step-10-b.png" width="450" />
+11. Verify your API permissions are correct and granted. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-11.png')} alt="ms-exchange-api-perm-step-11.png" width="800" />
+
+### Oauth 2.0 Generate Authorization Code
+
+:::note
+This authorization code will expire within 1 hour. Please setup your Sumo source before it expires.
+:::
+
+1. In a browser where you are already authenticated to your Azure portal at https://portal.azure.com, navigate to the URL below, but replace `{{client_id}}` with your `Application (client) ID` from the earlier steps.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:95]:
```suggestion
1. In a browser where you are already authenticated to your [Azure portal](https://portal.azure.com), navigate to the URL below, but replace `{{client_id}}` with your `Application (client) ID` from the earlier steps.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+10. **Secret** Enter your client secret generated within your Azure Application.
+
+11.  **Oauth 2.0 Authorization Code** Enter a valid authorization code. They expire within 1 hour of creation. The Microsoft Exchange Trace Logs Source will use this temporary code to automatically generate and rotate your API tokens for accessing the data.
+
+12. **Polling Interval Seconds** We recommend leaving this to its defaults. This value controls how often the Microsoft Exchange Trace Logs Source polls the the Microsoft APIs for new data.
+
+13. **Collection Time Offset** Use this value to offset the collection in the past by some amount of time. Microsoft claims data can be missing from the API within the last 24 hours. Set this to `86400` seconds aka 24 hours to help guarantee more complete results or reduce it if you are looking for more real time, but potentially incomplete data.
+
+14. When you are finished configuring the Source, click **Submit**.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:141]: Close spaces between numbering

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+9. **Directory (tenant) ID** Enter your tenant ID from your Azure Application. This should be a Globally Unique Identifier aka GUID.
+
+10. **Secret** Enter your client secret generated within your Azure Application.
+
+11.  **Oauth 2.0 Authorization Code** Enter a valid authorization code. They expire within 1 hour of creation. The Microsoft Exchange Trace Logs Source will use this temporary code to automatically generate and rotate your API tokens for accessing the data.
+
+12. **Polling Interval Seconds** We recommend leaving this to its defaults. This value controls how often the Microsoft Exchange Trace Logs Source polls the the Microsoft APIs for new data.
+
+13. **Collection Time Offset** Use this value to offset the collection in the past by some amount of time. Microsoft claims data can be missing from the API within the last 24 hours. Set this to `86400` seconds aka 24 hours to help guarantee more complete results or reduce it if you are looking for more real time, but potentially incomplete data.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:139]:
```suggestion
13. **Collection Time Offset**. Use this value to offset the collection in the past by some amount of time. Microsoft claims data can be missing from the API within the last 24 hours. Set this to `86400` seconds aka 24 hours to help guarantee more complete results or reduce it if you are looking for more real time, but potentially incomplete data.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+6. (Optional) **Fields.** Click the **+Add Field** link to define the fields you want to associate, each field needs a name (key) and value.
+
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema it is ignored, known as dropped.
+
+8. **Application (client) ID** Enter your client ID from your Azure Application. This should be a Globally Unique Identifier aka GUID.
+
+9. **Directory (tenant) ID** Enter your tenant ID from your Azure Application. This should be a Globally Unique Identifier aka GUID.
+
+10. **Secret** Enter your client secret generated within your Azure Application.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:133]:
```suggestion
10. **Secret**. Enter your client secret generated within your Azure Application.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+10.**Secret**. Enter your client secret generated within your Azure Application.
+
+11.**Oauth 2.0 Authorization Code** Enter a valid authorization code. They expire within 1 hour of creation. The Microsoft Exchange Trace Logs Source will use this temporary code to automatically generate and rotate your API tokens for accessing the data.
+
+12.**Polling Interval Seconds**. We recommend leaving this to its defaults. This value controls how often the Microsoft Exchange Trace Logs Source polls the the Microsoft APIs for new data.
+
+13.**Collection Time Offset**. Use this value to offset the collection in the past by some amount of time. Microsoft claims data can be missing from the API within the last 24 hours. Set this to `86400` seconds aka 24 hours to help guarantee more complete results or reduce it if you are looking for more real time, but potentially incomplete data.
+
+14.When you are finished configuring the Source, click **Submit**.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:140]: please close gaps between steps (delete lines in between)

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+### Create Client Secret
+
+1.Within your Azure application setup on the **App Registration** page, click **Certificates & secrets** from the left navigation pane and then click **New client secret**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-client-secret-step-1.png')} alt="ms-exchange-client-secret-step-1.png" width="700" />
+2.A right pane will slide out asking for a description and expiration for the secret. This secret is used by Sumo Logic to connect via OAuth 2.0 to establish continuous access to your Exchange Trace logs with an auto generated refresh token. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-client-secret-step-2.png')} alt="ms-exchange-client-secret-step-2.png" width="550" />
+3.Take note of the hidden value displayed on this page. After leaving this page, you will no longer be able to see this value from Azure. We recommend storing it in a protected password management vault. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-client-secret-step-3.png')} alt="ms-exchange-client-secret-step-3.png" width="700" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:57]: Add spaces between numbers and wording
```suggestion
1. Within your Azure application setup on the **App Registration** page, click **Certificates & secrets** from the left navigation pane and then click **New client secret**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-client-secret-step-1.png')} alt="ms-exchange-client-secret-step-1.png" width="700" />
2. A right pane will slide out asking for a description and expiration for the secret. This secret is used by Sumo Logic to connect via OAuth 2.0 to establish continuous access to your Exchange Trace logs with an auto generated refresh token. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-client-secret-step-2.png')} alt="ms-exchange-client-secret-step-2.png" width="550" />
3. Take note of the hidden value displayed on this page. After leaving this page, you will no longer be able to see this value from Azure. We recommend storing it in a protected password management vault. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-client-secret-step-3.png')} alt="ms-exchange-client-secret-step-3.png" width="700" />
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+6.Click **Roles and administrators|All roles** at the top of the page to view a full list of roles. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-6.png')} alt="ms-exchange-azure-roles-step-6.png" width="700" />
+7.Update your filter to Security Reader and click **Security Reader** text. Be sure to not click the checkbox next to **Security Reader** and instead click on the text name with the checkbox deselected. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-7.png')} alt="ms-exchange-azure-roles-step-7.png" width="500" />
+8.Click **Add assignments** for the Security Reader. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-8.png')} alt="ms-exchange-azure-roles-step-8.png" width="500" />
+9.Similar to the previous step, search and add the Azure application the role. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-9.png')} alt="ms-exchange-azure-roles-step-9.png" width="500" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:71]: Add spaces between numbers and wording

```suggestion
1. Click back on the main left navigation using the menu in the upper left corner of the screen and go back to the Azure Active Directory page. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-1.png')} alt="ms-exchange-azure-roles-step-1.png" width="400" />
2. Click **Roles and Administrators** in the left pane navigation of your Azure Active Directory page. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-2.png')} alt="ms-exchange-azure-roles-step-2.png" width="400" />
3. Search for the name Global Reader in the filter search bar and click **Global Reader** role text. Be sure to not click the checkbox next to **Global Reader** and instead click on the text name with the checkbox deselected. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-3.png')} alt="ms-exchange-azure-roles-step-3.png" width="400" />
4. Click **Add assignments** for the Global Reader.  <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-4.png')} alt="ms-exchange-azure-roles-step-4.png" width="350" />
5. **Add assignments** pane on the right allows you to search for your Azure application to assign the rights. Search for your application, select it and click **Add**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-5.png')} alt="ms-exchange-azure-roles-step-5.png" width="450" />
6. Click **Roles and administrators|All roles** at the top of the page to view a full list of roles. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-6.png')} alt="ms-exchange-azure-roles-step-6.png" width="700" />
7. Update your filter to Security Reader and click **Security Reader** text. Be sure to not click the checkbox next to **Security Reader** and instead click on the text name with the checkbox deselected. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-7.png')} alt="ms-exchange-azure-roles-step-7.png" width="500" />
8. Click **Add assignments** for the Security Reader. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-8.png')} alt="ms-exchange-azure-roles-step-8.png" width="500" />
9. Similar to the previous step, search and add the Azure application the role. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-azure-roles-step-9.png')} alt="ms-exchange-azure-roles-step-9.png" width="500" />
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+9.This time we are adding a similar permission as before. Click **Application permissions**, search for the text Reporting, check the checkbox next to `ReportingWebService.Read.All` and click **Add permissions**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-9.png')} alt="ms-exchange-api-perm-step-9.png" width="550" />
+10.You should now see both permissions, but you will need to click **Grant admin consent** to finalize the API permissions and confirm it. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-10-a.png')} alt="ms-exchange-api-perm-step-10-a.png" width="800" /> <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-10-b.png')} alt="ms-exchange-api-perm-step-10-b.png" width="450" />
+11.Verify your API permissions are correct and granted. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-11.png')} alt="ms-exchange-api-perm-step-11.png" width="800" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:87]: Add spaces between numbers and wording

```suggestion
1. Click back on the main left navigation using the menu in the upper left corner of the screen and go back to the **Azure Active Directory** page. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-1.png')} alt="ms-exchange-api-perm-step-1.png" width="500" />
2. You will see a menu of options for your Azure Active Directory on the left. Click **App registrations**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-2.png')} alt="ms-exchange-api-perm-step-2.png" width="400" />
3. Click on your Azure Application. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-3.png')} alt="ms-exchange-api-perm-step-3.png" width="500" />
4. Click **API permissions** in the left pane of your Azure application and click **Add a permission**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-4.png')} alt="ms-exchange-api-perm-step-4.png" width="450" />
5. A right pane slides out. Click **APIs my organization uses**, search for the text Office 365 Exchange Online, and click on the result. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-5.png')} alt="ms-exchange-api-perm-step-5.png" width="450" />
6. On the next page, click **Delegated permissions**, search for the text Reporting, check the checkbox next to `ReportingWebService.Read` and click **Add permissions**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-6.png')} alt="ms-exchange-api-perm-step-6.png" width="550" />
7. Now we need to add a second permission. Click **Add a permission** again. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-7.png')} alt="ms-exchange-api-perm-step-7.png" width="250" />
8. A right pane slides out. Click **APIs my organization uses**, search for the text Office 365 Exchange Online and click on the result. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-8.png')} alt="ms-exchange-api-perm-step-8.png" width="450" />
9. This time we are adding a similar permission as before. Click **Application permissions**, search for the text Reporting, check the checkbox next to `ReportingWebService.Read.All` and click **Add permissions**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-9.png')} alt="ms-exchange-api-perm-step-9.png" width="550" />
10. You should now see both permissions, but you will need to click **Grant admin consent** to finalize the API permissions and confirm it. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-10-a.png')} alt="ms-exchange-api-perm-step-10-a.png" width="800" /> <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-10-b.png')} alt="ms-exchange-api-perm-step-10-b.png" width="450" />
11. Verify your API permissions are correct and granted. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-api-perm-step-11.png')} alt="ms-exchange-api-perm-step-11.png" width="800" />
```

**Review by kimsauce — DISMISSED** [review]
formatting

@kimsauce All suggestions are implemented and resolved.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+2.Use the left navigation pane (might be hidden under the menu icon) and select **Azure Active Directory**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-app-reg-step-2.png')} alt="ms-exchange-app-reg-step-2.png" width="400" />
+3.You will see a menu of options for your Azure Active Directory on the left. Click **App registrations**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-app-reg-step-3.png')} alt="ms-exchange-app-reg-step-3.png" width="300" />
+4.Click **New registration** <br/> <img src={useBaseUrl('img/send-data/ms-exchange-app-reg-step-4.png')} alt="ms-exchange-app-reg-step-4.png" width="550" />
+5.Enter the required information as shown in the screenshot. Click **Register**. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-app-reg-step-5.png')} alt="ms-exchange-app-reg-step-5.png" width="600" />
+6.You are redirected to the overview page of your newly created application. Take note of the **Application (client) ID** and the **Directory (tenant) ID** as they will be required later. <br/> <img src={useBaseUrl('img/send-data/ms-exchange-app-reg-step-6.png')} alt="ms-exchange-app-reg-step-6.png" width="500" />
```
[the remark this answers is not in the document]
Comment (JainM6) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md:51]: Your suggestion was to close spaces between numbers, so I did that. The reason was not understood for sure. Maybe the suggestion was not clear to me.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md (hunk 7 above)
Comment (JainM6) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md]: DONE

**Review by nmohantyctr — APPROVED** [review]

**Review by nmohantyctr — APPROVED** [review]

**Comment by JainM6** [comment]
> formatting
> 
> @kimsauce All suggestions are implemented and resolved.
All suggestions are implemented and resolved.

**Pull request merged**